code/modules/client/preference_setup/loadout/lists/storage.dm

```diff
@@ -3,71 +3,6 @@
 	category = /datum/gear/storage/
 	slot = slot_tie
 
-/datum/gear/storage/brown_vest
```

Comment:
i dont see a reason to remove this webbing and pouches

Reply:
Webbing is reserved for engineers.